### PR TITLE
Add onnx.Abs lowering to TOSA

### PR DESCRIPTION
**Pull request by roberteg16**

### CODE REVIEW

File: src/Conversion/ONNXToTOSA/Math/Elementwise.cpp [hunk 1]
```diff
@@ -31,6 +31,20 @@ struct TOSADialectOp<ONNXNegOp> {
...
 };
 
+struct AbsIOSupportedTypes {
+  static LogicalResult checkType(
+      ConversionPatternRewriter &rewriter, Type scalarType, Operation *op) {
+    if (!mlir::isa<mlir::BFloat16Type, mlir::Float16Type, mlir::Float32Type>(
+            scalarType) &&
+        !scalarType.isSignlessInteger(/*width=*/32)) {
+      return rewriter.notifyMatchFailure(op,
+          "this operation only supports singless 32 integer or fp16, fp32"
```
Comment (ttjost) [inline src/Conversion/ONNXToTOSA/Math/Elementwise.cpp:41]:
```suggestion
          "this operation only supports signless 32 integer or fp16, fp32"
```

Comment (ttjost) [reply src/Conversion/ONNXToTOSA/Math/Elementwise.cpp:41]: Maybe you can commit directly. I'll let you do it. ;)

**Review by ttjost — APPROVED** [review]
LGTM.

File: src/Conversion/ONNXToTOSA/Math/Elementwise.cpp (hunk 1 above)
Comment (roberteg16) [reply src/Conversion/ONNXToTOSA/Math/Elementwise.cpp]: Done

**Pull request merged**